libs/community/langchain_community/vectorstores/opensearch_vector_search.py

```diff
@@ -1026,6 +1061,48 @@ def _raw_similarity_search_with_score_by_vector(
                 vector_field,
                 score_threshold=score_threshold,
             )
+        elif search_type == HYBRID_SEARCH:
+            keyword_weight, vector_weight = self.hybrid_search_weights
```

Comment:
hi @AndreasThinks , should these weights be got from kwargs? As I see in your changed docs, this param was passed at search method rather than the store initiation

Reply:
yes, fair point, that would probably make it more consistent with approach - shall make the changes.